docs/tutorials/2D-parabolic.rst

```diff
@@ -1,10 +1,15 @@
-2D parabolic dune model
+Parabolic dune model
```

Comment:
The parabolic dune example can't be run in a few minutes by users on their laptops.

```
        Time elapsed / Total time / Time remaining / Average Timestep
000.0%       0:00:03 / 1 day, 6:37:52 / 1 day, 6:37:49 / 3600.0
```

@Sierd , we need to remove or replace this example with another one that can be run quickly by users. Do you have a replacement in mind?

Reply:
We agree on keeping the parabolic dune example because trying out examples with different run times will help give users a clearer picture of simulating different models with aeolis.